contracts/connectors/dydxFlash/main.sol

```diff
@@ -33,6 +33,6 @@ abstract contract FlashLoanResolver is DSMath, Basic, Events {
     }
 }
 
-contract ConnectDydxFlashLoan is FlashLoanResolver {
+contract ConnectV2DydxFlashLoan is FlashLoanResolver {
```

Comment:
@mubaris I don't know think we need this, we can a new folder and move into that.

Reply:
Deleted this as we already have this in `connectors_old`